backend/database_handler/models.py

```diff
@@ -57,10 +57,10 @@ class Transactions(Base):
     __tablename__ = "transactions"
     __table_args__ = (
         CheckConstraint("type = ANY (ARRAY[0, 1, 2])", name="transactions_type_check"),
-        PrimaryKeyConstraint("id", name="transactions_pkey"),
+        PrimaryKeyConstraint("hash", name="transactions_pkey"),
     )
 
-    id: Mapped[int] = mapped_column(Integer, primary_key=True, init=False)
+    hash: Mapped[str] = mapped_column(String(255), primary_key=True, unique=True)
```

Comment:
nit: from my tests, all transaction hashes have 66 ("0x" + 64) characters, it'd be nice to limit that here too

Reply:
Replaced 255 by 66 in models: transactions.hash, transaction_audits.transaction_hash, as well as in migration file, please review